accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/configreader/ConfigFilesRetrieverImpl.java

```diff
@@ -76,8 +77,22 @@
         return configs;
     }
 
-    static boolean isRelevantConfiguration(final String entryName, final String parentName, ExtendedSlingSettingsService slingSettings) {
-        if (!entryName.endsWith(".yaml") && !entryName.equals("config") /* name 'config' without .yaml allowed for backwards compatibility */) {
+    static boolean isRelevantConfiguration(final PackageEntryOrNode entry, final String parentName,
+                                           ExtendedSlingSettingsService slingSettings, Collection<String> configFilePatterns) throws Exception {
+        boolean matchPattern = configFilePatterns.isEmpty();
+        for (String pathPattern : configFilePatterns) {
+            if (entry.getPath().matches(pathPattern)) {
+                matchPattern = true;
+            }
+        }
+        if (!matchPattern) {
+            LOG.info("Skipped file '{}' because it didn't match any path pattern", entry.getPath());
+            return false;
+        }
+        String entryName = entry.getName();
+        if (!entryName.endsWith(".yaml") && !entryName.equals("config") /*
```

Comment:
weird linebreaks...

Reply:
moved the comment a line above to fix the issue